src/porepy/models/fluid_property_library.py

```diff
@@ -0,0 +1,375 @@
+""""Module containing some constant heuristic fluid property implemenations.
```

Comment:
This file takes fluid-related heuristics out of the `constitutive_laws.py` file.
Some methods are reworked to fit the general fluid concept, where properties of phases are dynamically created based on the number of modelled phases.